index.html

```diff
@@ -15,6 +15,13 @@
 	<body>
 		<div id="header">
 			<img id="header-logo" src="t-logo.png"/>
+			<div id="navbar">
```

Comment:
instead of `id="navbar"` you can use `nav`
```suggestion
			<nav">
```